cloudaux/openstack/decorators.py

```diff
@@ -5,26 +5,25 @@
     :license: Apache, see LICENSE for more details.
 .. moduleauthor:: Michael Stair <mstair@att.com>
 """
+import os
 from functools import wraps
 
-from os_client_config import OpenStackConfig
-from openstack import connection
+from openstack import connect
+from openstack.config.loader import OpenStackConfig
 from openstack.exceptions import HttpException
 
 """ this is mix of the aws and gcp decorator conventions """
 
 CACHE = {}
 
 def _connect(cloud_name, region, yaml_file):
-    occ = OpenStackConfig(config_files=[yaml_file])
-    cloud = occ.get_one_cloud(cloud_name, region_name=region)
-    _cloud_name = cloud.get_auth_args().get('project_id')
-    return ( _cloud_name, connection.from_config(cloud_config=cloud) )
+    os.environ["OS_CLIENT_CONFIG_FILE"] = yaml_file
```

Comment:
FYI, need to set this to be able to set an absolute path to a clouds.yaml file and override the default search paths.  https://docs.openstack.org/openstacksdk/latest/user/config/configuration.html#config-files